### PR TITLE
feat(mobile): Welcome screen bb-183

### PR DESCRIPTION
**Pull request by Shcracoziabra**
### Purpose
Create mobile Welcome screen component.

### Overview
Updated BaseColor enum with DARK_BLUE color to use as screen text background color.
Added Wecome screen with empty handleContinuePress function.

### Screenshots
welcome screen

### CODE REVIEW

File: apps/mobile/src/navigations/root/root.tsx [hunk 1]
```diff
@@ -29,6 +30,10 @@ const Root: React.FC = () => {
 				/>
 			) : (
 				<NativeStack.Group>
+					<NativeStack.Screen
+						component={Welcome}
+						name={RootScreenName.WELCOME}
+					/>
```
Comment (fshabanov) [inline apps/mobile/src/navigations/root/root.tsx:36]: I believe we should have this screen when the user is logged it

**Review by fshabanov — CHANGES_REQUESTED** [review]

File: apps/mobile/src/navigations/root/root.tsx (hunk 1 above)
Comment (revenets) [reply apps/mobile/src/navigations/root/root.tsx:36]: Regarding the design this screen should be after Onboarding questions. We'll have a separate ticket for screens order later.

**Review by revenets — APPROVED** [review]

File: apps/mobile/src/navigations/root/root.tsx (hunk 1 above)
Comment (revenets) [reply apps/mobile/src/navigations/root/root.tsx:36]: Didn't notice that it is in the Auth group. Yes, please move it above

**Review by fshabanov — APPROVED** [review]

**Comment by fshabanov** [comment]
Merge the main branch

**Pull request merged**